pkg/driver_cryptocell/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME=driver_cryptocell
+PKG_URL=https://github.com/Einhornhool/RIOT-cryptocell-driver.git
```

Comment:
This package points to an archive (.a) only, with headers repository. That's quite limited: only nrf52840 cpu is supported. Is there a possibility and a plan to provide an open-source implementation ?

Also note that the package is too much limited IMHO: it can only be built for nrf52840dk board but I think any nrf52840 CPU based board should work with it.

Reply:
Unfortunately this driver is closed source and there don't seem to be any plans to make it open source. 
Nordic only provides the archives and header files. 

The limitation is my mistake, I did not realize that the nrf52840 also had an accelerator, I will fix that.